docs/destinations/warehouses-and-lakes/index.md

```diff
@@ -4,7 +4,40 @@ sidebar_position: 2
 ---
 
 ```mdx-code-block
-import DocCardList from '@theme/DocCardList';
-
-<DocCardList />
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 ```
+
+To store your enriched Snowplow data, you will need to determine which loader to set-up to ensure your enriched data reaches your data warehouse or lake. 
+
+### Selecting a loader for your data warehouse
+#### Selecting a loader by destination and cloud provider
+<Tabs groupId="destination">
+  <TabItem value="aws" label="AWS" default>
+
+| Destination | Loader | Status |
+| --- | --- | --- |
+| Redshift | [Snowplow RDB Loader](/docs/destinations/warehouses-and-lakes/rdb/index.md) | Production-ready |
+| Snowflake | [Snowplow RDB Loader](/docs/destinations/warehouses-and-lakes/rdb/index.md) | Production-ready |
+| Databricks | [Snowplow RDB Loader](/docs/destinations/warehouses-and-lakes/rdb/index.md) | Production-ready |
+| Postgres | [Postgres Loader](/docs/destinations/warehouses-and-lakes/postgres/index.md) | Early release |
+
+  </TabItem>
+  <TabItem value="gcp" label="GCP">
+
+| Destination | Loader | Status |
+| --- | --- | --- |
+| Snowflake | [Snowplow RDB Loader](/docs/destinations/warehouses-and-lakes/rdb/index.md) | Production-ready |
+| BigQuery | [BigQuery Loader](/docs/destinations/warehouses-and-lakes/bigquery/index.md) | Production-ready |
+| Postgres | [Postgres Loader](/docs/destinations/warehouses-and-lakes/postgres/index.md) | Early release |
+
+  </TabItem>
+</Tabs>
+
+#### If you are not using Kinesis or Pub/Sub
+
+Other stream processing technologies are supported by the [Stream Collector](/docs/pipeline-components-and-applications/stream-collector/index.md) and [Enrich](/docs/pipeline-components-and-applications/enrichment-components/index.md), but if you prefer to use a stream processing technology other than Kinesis or Pub/Sub, currently you will need to define your own process to load your enriched data into your data warehouse.
```

Comment:
I would make this a note rather than a choice point:

```suggestion
:::note

Our warehouse loaders pick up the data from [Enrich Kinesis](...) on AWS and [Enrich Pubsub](...) on GCP. Both [Stream Collector](/docs/pipeline-components-and-applications/stream-collector/index.md) and [Enrich](/docs/pipeline-components-and-applications/enrichment-components/index.md) allow you to use a streaming technology other than Kinesis and Pub/Sub, but if you go that route, you will need to define your own process to load your enriched data into the warehouse.

:::
```

Reply:
Good idea. Will do.